hoot-core-test/src/test/cpp/hoot/core/schema/ScoreMatrixTest.cpp

```diff
@@ -51,7 +53,7 @@ class ScoreMatrixTest : public CppUnit::TestFixture
      * California   | 0.8          | 0.0        | 0.98   | 1   | 0.99
      */
 
-    ScoreMatrix uut(5, 3);
+    ScoreMatrix<double> uut(5, 3);
```

Comment:
Changed the `ScoreMatrix` class to a template class because I wanted a class of `long` instead of `double`.